.github/workflows/node-ci.yml

```diff
@@ -128,6 +112,11 @@ jobs:
         # Build production
         build-script: "build:prodsizecheck"
 
+      - uses: Khan/actions@check-for-changeset-v0
```

Comment:
What does this do for the Version Packages PR that changeset creates? Does it just ignore it?

Reply:
yeah, the check-for-changeset action checks if the branch name is `changeset-release/main` and ignores it.